oneview_redfish_toolkit/multiple_oneview.py

```diff
@@ -235,6 +237,34 @@ def execute_query_ov_client(ov_client, resource, function, *args, **kwargs):
     """Execute query for resource on OneView client received as parameter"""
     ov_resource = getattr(ov_client, resource)
     ov_function = getattr(ov_resource, function)
+    """
+    # below two condition are checked due to changes made in oneview sdk
```

Comment:
Is the whole block commented out ?

Reply:
I missed to delete this block. No longer required